tests/common.go

```diff
@@ -2713,13 +2713,13 @@ func DeleteCluster(name string, orgID string, ctx context1.Context) error {
 }
 
 // DeleteBackupLocation deletes backup location
-func DeleteBackupLocation(name string, backupLocationUID string, orgID string) error {
+func DeleteBackupLocation(name string, backupLocationUID string, orgID string, DeleteExistingBackups bool) error {
```

Comment:
To ensure consistency, perhaps we could consider renaming it to "backupLocationName"? Although it may not be of significant importance.

Reply:
This is outside the scope of this PR we can do it in the next PR.